x-pack/plugins/graph/public/app.js

```diff
@@ -730,30 +759,46 @@ app.controller('graphuiPlugin', function ($scope, $route, $interval, $http, kbnU
   $scope.topNavMenu = [];
   $scope.topNavMenu.push({
     key: 'new',
-    description: 'New Workspace',
-    tooltip: 'Create a new workspace',
+    description: i18n('xpack.graph.topNavMenu.newWorkspaceAriaLabel', {
+      defaultMessage: 'New Workspace',
+    }),
+    tooltip: i18n('xpack.graph.topNavMenu.newWorkspaceTooltip', {
+      defaultMessage: 'Create a new workspace',
+    }),
     run: function () {canWipeWorkspace(function () {kbnUrl.change('/home', {}); });  },
   });
   if (!$scope.allSavingDisabled) {
     $scope.topNavMenu.push({
       key: 'save',
-      description: 'Save Workspace',
-      tooltip: 'Save this workspace',
+      description: i18n('xpack.graph.topNavMenu.saveWorkspace.enabledAriaLabel', {
```

Comment:
Also it looks like it is description, not label

Reply:
It is an aria-label on UI